### DISCUSSION TITLE
compartir carpeta con los las terminales

### DISCUSSION THREAD

**Discussion by mcrobot-lujan** [ltsp/ltsp#535] [General]
necesitaria compartir una carpeta que esta en /compartidos en el server
en las terminales solo veo la carpeta vacia
***********************
I would need to share a folder that is in / shared on the server
in the terminals I only see the empty folder

**Comment by alkisg** [comment]
How is /folder mounted on the server?
What kind of content does it have?

1. Static: you can include it in the image, e.g. `ltsp image /,,/folder,subdir=folder`
2. Dynamic, read-only: you can share it via NFS-ro, see FSTAB_x in ltsp.conf
3. Dynamic, read-write: you can share it via NFS-rw, see FSTAB_x in the ltsp.conf man page
